src/gui/rss/rsswidget.cpp

```diff
@@ -130,6 +130,19 @@ RSSWidget::RSSWidget(IGUIApplication *app, QWidget *parent)
     m_ui->rssDownloaderBtn->setIcon(UIThemeManager::instance()->getIcon(u"downloading"_s, u"download"_s));
 #endif
 
+    m_rssFilter = new LineEdit(this);
+    m_rssFilter->setObjectName("rssFilter");
+    QSizePolicy sizePolicy {QSizePolicy::Policy::Fixed, QSizePolicy::Policy::Fixed};
+    sizePolicy.setHorizontalStretch(200);
+    sizePolicy.setVerticalStretch(0);
+    sizePolicy.setHeightForWidth(m_rssFilter->sizePolicy().hasHeightForWidth());
+    m_rssFilter->setSizePolicy(sizePolicy);
```

Comment:
I don't think it needs a size policy. I'll test it and report what I find later.

Reply:
You don't need minimum size, it will be provided by default.
If you want to limit the maximum size use https://doc.qt.io/qt-6/qwidget.html#maximumWidth-prop :
```suggestion
    m_rssFilter->setMaximumWidth(200);
```